include/swift/Runtime/Backtrace.h

```diff
@@ -123,8 +134,15 @@ struct BacktraceSettings {
 
 SWIFT_RUNTIME_STDLIB_INTERNAL BacktraceSettings _swift_backtraceSettings;
 
-SWIFT_RUNTIME_STDLIB_SPI SWIFT_CC(swift) bool _swift_isThunkFunction(const char *mangledName);
+SWIFT_RUNTIME_STDLIB_SPI
+bool _swift_backtrace_isThunkFunction(const char *mangledName);
 
+SWIFT_RUNTIME_STDLIB_SPI
```

Comment:
Would this benefit from a doc comment that clarifies the use of each parameter and the return pointer as well?

Reply:
There's a fairly extensive doc comment for it in the `.cpp` file already; I'd rather not duplicate it here, and as this is SPI anyway I'm not especially fussed about it being here versus there.

Reply:
(This is also not really the right PR to review this particular code :-))